Displayed brand info

YuliiaMilian opened the pull request.

YuliiaMilian committed:
Displayed brand info

Khousheish reviewed (approved):
Everythink looks good. I really like how you used both approaches with .then but another one with async/await.
If you want you can take it a step further and figure out how to display the colors and the logos being returned from the brandfetch api.

YuliiaMilian committed:
Displayed brand colors and image

YuliiaMilian commented:
@Khousheish Thanks for your feedback! I added colors to the text) Regarding the image I added that, but unfortunately, I made to many requests and couldn't check it( It is just about my logical thinking I believe the image should displayed.

YuliiaMilian committed:
Fixed brand image
